bundles/specmate-dbprovider-h2/src/specmate/dbprovider/h2/config/H2ProviderConfig.java

```diff
@@ -17,10 +17,8 @@
 public class H2ProviderConfig {
 
 	public static final String PID = "com.specmate.dbprovider.h2.H2ProviderConfig";
-	public static final String KEY_JDBC_CONNECTION = "cdoJDBCConnection";
-	public static final String KEY_REPOSITORY_NAME = "cdoRepositoryName";
-	public static final String KEY_RESOURCE_NAME = "cdoResourceName";
-	private static final String DB_PREFIX = "h2.";
+	public static final String KEY_JDBC_CONNECTION = "h2.jdbcConnection";
+	public static final String KEY_REPOSITORY_NAME = "cdo.repositoryName";
```

Comment:
repository name is, IMO, not needed anymore in this config. That should be managed in the SpecmateCDOServerConfig